### ISSUE TITLE
Ranning Error

### ISSUE THREAD

**Issue by Yuhan291** [zhengchen1999/DAT#26]
What should I do if I meet this problem?
"No object named 'DATModel' found in 'model' registry!"

Thank you

**Comment by zhengchen1999** [comment]
You can refer to #1, which is a similar question.

**Comment by Yuhan291** [comment]
Thank you for your answer.

I meet another problem like this:
ModuleNotFoundError: No module named 'six'

What is a 'six' module?

**Comment by zhengchen1999** [comment]
Sorry, I don't know. May be some library is dependent on it.
Run `pip install six` to install it.

PS: If you create **a new library from scratch** (https://github.com/zhengchen1999/DAT?tab=readme-ov-file#dependencies), similar problems should not occur.

**Comment by zhengchen1999** [comment]
Yes, [this](https://github.com/zhengchen1999/DAT/blob/main/options/Train/train_DAT_x2.yml#L85) is the frequency.
